src/sugar3/graphics/Makefile.am

```diff
@@ -16,6 +16,7 @@ sugar_PYTHON =              \
 	palettemenu.py          \
 	palettewindow.py        \
 	panel.py                \
+        popwindow.py            \
```

Comment:
Probably mismatch of tab and space characters in this line compared to the lines either side, causing diff to show slight indentation.